Configurations/10-main.conf

```diff
@@ -1925,7 +1927,7 @@ my %targets = (
         #as               => "I4S",
         #debug_aflags     => "/NOOPTIMIZE/DEBUG",
         #release_aflags   => "/OPTIMIZE/NODEBUG",
-        bn_opts          => "SIXTY_FOUR_BIT RC4_INT",
+        bn_ops           => "SIXTY_FOUR_BIT RC4_INT",
     },
     "vms-ia64-p32" => {
         inherit_from     => [ "vms-generic" ],
```

Comment:
So inheriting from vms-ia64 is not an option? I mean wouldn't it be more natural if vms-ia64-p32 and vms-ia64-p64 inherited from vms-ia64, [which in turn inherits from vms-generic]?

Reply:
It would, but because `vms_info` caches results for whatever pointer size it's given first, it becomes problematic.  I've too many things going on to think about that particular detail for the moment...